### ISSUE TITLE
Two fields in the advanced search form are missing labels

### ISSUE THREAD

**Issue by sandbergja** [projectblacklight/blacklight#3072]
### Steps to recreate

In Firefox:

1. Go to https://demo.projectblacklight.org/catalog/advanced
2. Right click on the "all" dropdown in the sentence "Match all of the fields below".
3. Select "Inspect Accessibility Properties"
4. Note that the name is "null", and that you get an alert "Form elements must be labeled".  Expected behavior is that the form element does have an accessible name.
5. Repeat steps 2-4 on the "relevance" sort dropdown at the end of the form.

### Suggested fixes

* Add an aria-label to the first dropdown
* Associate the "relevance" dropdown with the "Sort results by" label, perhaps using aria-labelledby.

**Comment by sandbergja** [comment]
Just realized that @cbeer already fixed these in #3068.  Closing this issue.
